EventFilter/EcalRawToDigi/interface/DeclsForKernels.h

```diff
@@ -0,0 +1,55 @@
+#ifndef EventFilter_EcalRawToDigi_interface_DeclsForKernels_h
+#define EventFilter_EcalRawToDigi_interface_DeclsForKernels_h
```

Comment:
Will this header be needed outside of this package? Or in `src` in addition to `plugins`? If no, please (eventually) move this file to `plugins`.

Reply:
yeah, this guy gets included into .cu where kernels + host wrapers for kernels are

Reply:
and .cu are sitting in the lib (/src)

Reply:
Are the kernels intended to be used by other packages?

Reply:
nope, only edm plugins from that package use that lib. will move that into plugins/